sqs2http.go

```diff
@@ -106,13 +97,15 @@ func main() {
 	// listen for kill/term/stop signals from user/os
 	stop.Listen()
 
+	var wg sync.WaitGroup
+
 	// create the workers
 	for tmp > 0 {
 		// spawn a goroutine for each worker
 		go func() {
-			empties := uint64(0) //<< keeps track of number of subsequent empty replies from queue
-			bo, err := expbo.New(uint64(1000), uint64(*boMax) * 1000, 2) //<< exponential backoff
-
+			defer wg.Done()
```

Comment:
I kinda forget how wait groups work, but I think you need to do something like `wg.Add(1)` or something, right?

Reply:
lol true, forgot this